CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+**Features**:
+
+- Normalize old RN SDK app start time measurements and spans. ([#2358](https://github.com/getsentry/relay/pull/2358))
```

Comment:
Could we, please, use the full name of the SDK? Since it's not really clear what `RN`  stands for?

```suggestion
- Normalize old React Native SDK app start time measurements and spans. ([#2358](https://github.com/getsentry/relay/pull/2358))
```